reservoirpy/utils/sklearn_helper.py

```diff
@@ -0,0 +1,171 @@
+from functools import wraps
+from typing import Callable
+
+import numpy as np
+try: #pragma: no cover
+    from sklearn import linear_model
+except ImportError:
+    raise ImportError("The sklearn library is required for this module.")
+
+
+def get_linear(method) -> Callable:
+    """
+    Returns a scikit-learn linear model class given its method as a string.
+
+    Parameters
+    ----------
+    method : str
+        The method of the scikit-learn linear model.
+
+    Returns
+    -------
+    Callable
+        The scikit-learn linear model class.
+    """
+    return getattr(linear_model, method)
+
+
+class TransformInputSklearn(object):
+    def _for_regression(self, X, y):
```

Comment:
Maybe a naive remark, but could these two components (TranformInputSklearn and TransformOutputSklearn) be directly integrated to the SklearnNode? Like they could be applied to custom `run` or `fit` methods just before calling the standard base class methods for instance

Reply:
Maybe I am wrong, but since the transform functions are only used for sklearn and I have explained the logic behind having the transform function in the tutorial, maybe we can keep it as it is?